Classes/Backend/IndexingConfigurationSelectorField.php

```diff
@@ -215,9 +215,13 @@ protected function renderSelectCheckbox($items, $selectedValues) {
 			)
 		);
 
-		$selectFieldRenderer = $formEngine = GeneralUtility::makeInstance('TYPO3\\CMS\\Backend\\Form\\FormEngine');
-		if (!method_exists($selectFieldRenderer, 'getSingleField_typeSelect_checkbox')) {
-			if (class_exists('TYPO3\\CMS\Backend\\Form\\Element\\SelectElement')) {
+		$selectFieldRenderer = $formEngine = NULL;
+		if (class_exists('TYPO3\\CMS\\Backend\\Form\\FormEngine')) {
+			$selectFieldRenderer = $formEngine = GeneralUtility::makeInstance('TYPO3\\CMS\\Backend\\Form\\FormEngine');
+		}
+
+		if (!isset($selectFieldRenderer) || !method_exists($selectFieldRenderer, 'getSingleField_typeSelect_checkbox')) {
+			if (class_exists('TYPO3\\CMS\Backend\\Form\\Element\\SelectElement') && isset($formEngine)) {
```

Comment:
Thx. I will change this later. Althought because of the single ' it doesn't really matter. It would still work but CGL is two.